test/test_jit_fuser.py

```diff
@@ -282,6 +282,22 @@ def funcOptMax(a, b):
             graph = backward_graph(s)
             self.assertAllFused(graph)
 
+    @unittest.skipIf(IS_WINDOWS, "NYI: fuser support for Windows")
+    @unittest.skipIf(not RUN_CUDA, "fuser requires CUDA")
+    @skipIfRocm
+    def test_dropout(self):
+        def func(x):
+            x = torch.nn.functional.dropout(x)
+            return torch.nn.functional.relu(x)
+
+        a = torch.randn(4, 4, dtype=torch.float, device='cuda', requires_grad=True)
+        s = torch.jit.script(func, (a,))
+        self.assertAllFused(s.graph_for(a,), except_for={'aten::div', 'prim::Constant'})
```

Comment:
`aten::div` is a fusable op right? why it does not get fused?

Reply:
It's division of a scalar argument, no point in fusing.